hieradata_aws/common.yaml

```diff
@@ -806,6 +807,8 @@ grafana::dashboards::deployment_applications:
     show_slow_requests: false
   contacts:
     docs_name: 'contacts-admin'
+  content-audit-tool:
+    # logstasher >1.x
```

Comment:
Is there any reason for adding/not-adding:
```
fields_prefix: ''
```
?

Reply:
Added for consistency and fixed up.